kotlin/internal/jvm/compile.bzl

```diff
@@ -342,6 +342,12 @@ def _kotlinc_options_provider_to_flags(opts):
         flags.append("-nowarn")
     if opts.x_use_experimental:
         flags.append("-Xuse-experimental=kotlin.Experimental")
+    if opts.x_use_ir:
+        flags.append("-Xuse-ir")
```

Comment:
That's it? We don't need extra libs or some such?

Reply:
The extra libs that are needed can be provided via compiler plugins

Reply:
For IR or compose?

Reply:
For compose. The new IR backend is just the new compiler backend that Jetbrains has been working on. Jetpack Compose is a compiler plugin that was built to leverage the new backend (enabled via `-Xplugin=...` or `kt_compiler_plugin(...)`.